test/e2e-common.sh

```diff
@@ -357,7 +357,7 @@ function install() {
   echo "serving post-install config at ${ytt_post_install_result}"
 
   run_kapp deploy --yes --app serving --file "${ytt_result}" \
-        || fail_test "failed to setup knative"
+        || kubectl logs deploy/activator -n ${SYSTEM_NAMESPACE}
```

Comment:
Assuming this was for debugging?

Also kapp has a logs feature that you can use for debugging

```
Logs Flags:
      --logs      Show logs from Pods annotated as 'kapp.k14s.io/deploy-logs' (default true)
      --logs-all  Show logs from all Pods
```

Reply:
Yepp, forgot to remove.